tests/e2e/e2e_experimental_test.go

```diff
@@ -0,0 +1,50 @@
+//go:build experimental
+// +build experimental
```

Comment:
let's remove experimental from here, we already lifted ibc quota out of experimental build.